docs/Contributing.md

```diff
@@ -82,15 +82,17 @@ Please do:
 - **DO** submit all code changes via pull requests (PRs) rather than through a
   direct commit. PRs will be reviewed and potentially merged by the repo
   Committers after a peer review that includes at least one Committer.
-- **DO** give PRs short-but-descriptive names (e.g. "Improve code coverage for
-  System.Console by 10%", not "Fix #1234").
+- **DO** give PRs short, but descriptive names (e.g. "Improve code coverage for
```

Comment:
I don't think you need the comma between the adjectives, it's only used before "but" when introducing an independent clause, which "but descriptive names" is not.